scripts/upload_fake_passwordsBB.py

```diff
@@ -0,0 +1,209 @@
+#
+#  This is a little script to populate Firefox Sync with
+#  fake password records.  Use it like so:
+#
+#    $> pip install PyFxA syncclient cryptography
+#    $> python ./upload_fake_passwords.py 20
+#
+#  It will prompt for your Firefox Account email address and
+#  password, generate and upload 20 fake password records, then
+#  sync down and print all password records stored in sync.
+#    
+
+import os
+import time
+import json
+import random
+import string
+import getpass
+import hmac
+import hashlib
+import base64
+import uuid
+from binascii import hexlify
+
+from cryptography.hazmat.primitives.ciphers import Cipher, algorithms, modes
+from cryptography.hazmat.primitives import padding
+from cryptography.hazmat.backends import default_backend
+
+import fxa.core
+import fxa.crypto
+import syncclient.client
+
+
+CRYPTO_BACKEND = default_backend()
+
+
+EMAIL = "firefoxlockbox@gmail.com"
+PASSWORD = "aabbcc112233!"
```

Comment:
Could we get these credentials from an environment variable set in BB?

Reply:
Yes, we can have both!